server/app/controllers/admin/ProgramAdminManagementController.java

```diff
@@ -85,6 +85,36 @@ public Result update(Http.Request request, long programId) {
     }
   }
 
+  /** Removes `adminEmail` as a program admin for the program identified by `programId`. */
+  @Secure(authorizers = Authorizers.Labels.CIVIFORM_ADMIN)
+  public Result delete(Http.Request request, long programId, String adminEmail) {
+    try {
+      roleService.removeProgramAdmins(programId, ImmutableSet.of(adminEmail));
+      return redirect(routes.AdminProgramController.edit(programId));
```

Comment:
This will redirect to the program edit view. I think you want:
```
return ok(routes.ProgramAdminManagementController.edit(programId);
```

Reply:
As discussed via Slack, redirecting to the new (correct) view that you identified.